src/prism-formatted/utils.ts

```diff
@@ -0,0 +1,47 @@
+const langs: { [key: string]: string } = {
+	js: "JavaScript",
+	javascript: "JavaScript",
+	css: "CSS",
+	html: "HTML",
+	python: "python",
+	py: "python",
+	xml: "XML",
+	jsx: "JSX",
+	scss: "SCSS",
+	sql: "SQL",
+	http: "HTTP",
+	json: "JSON",
+	pug: "pug",
+};
+
+// Extracts the programming language name from the provided string.
+export const getCodeLanguageFromString = (text: string) => {
+	const matches = text.match(/(?:class="language-)(\w*)(?:")/);
```

Comment:
Rather than leaning on regex to find the className, why not pass in a callback that takes `codeName` and returns the label? That way we can make use of the fact we have direct access to `className` inside `enhancePrismAccessibility` to get `codeName`.

Then, in /learn, we can pass in 

```js
(codeName) => i18next.t('aria.code-example', { codeName })
```